client_test.go

```diff
@@ -88,3 +88,46 @@ func TestClient_CreateToken(t *testing.T) {
 		})
 	}
 }
+
+func TestSendUserCustomEvent(t *testing.T) {
+	c := initClient(t)
+
+	tests := []struct {
+		name         string
+		event        *UserCustomEvent
+		targetUserID string
+		expectedErr  string
+	}{
+		{
+			name: "ok",
+			event: &UserCustomEvent{
+				Type: "custom_event",
```

Comment:
could you add some custom data to test marshaling?

Reply:
I can but I am not sure we have a way to ensure that the data is marshalled correctly. We can only check that `Marshal` doesn't return an error. It's probably enough though.

Reply:
Yup, checking error is better and there could be another test, just exercises marshal/unmarshal exclusively.

Reply:
Another idea is to marshal and unmarshal and check if it was done correctly

Reply:
It seems what you did